server/sonar-server/src/test/java/org/sonar/server/qualitygate/ws/CreateActionTest.java

```diff
@@ -59,26 +62,55 @@
   private WsActionTester ws = new WsActionTester(underTest);
 
   @Test
-  public void create_quality_gate() {
-    logInAsQualityGateAdmin();
+  public void create_quality_gate_without_parameter() {
+    logInAsQualityGateAdmin(db.getDefaultOrganization());
 
-    CreateResponse response = executeRequest("Default");
+    String qgName = "Default";
+    CreateResponse response = executeRequest(Optional.empty(), qgName);
 
-    assertThat(response.getName()).isEqualTo("Default");
+    assertThat(response.getName()).isEqualTo(qgName);
     assertThat(response.getId()).isNotNull();
     dbSession.commit();
-    QualityGateDto qualityGateDto = dbClient.qualityGateDao().selectByName(dbSession, "Default");
+
+    QualityGateDto qualityGateDto = dbClient.qualityGateDao().selectByOrganizationAndName(dbSession, db.getDefaultOrganization(), qgName);
+    assertThat(qualityGateDto).isNotNull();
+  }
+
+  @Test
+  public void create_quality_gate_with_organization() {
+    OrganizationDto organizationDto = organizationDbTester.insert();
+    logInAsQualityGateAdmin(organizationDto);
+
+    String qgName = "Default";
+    CreateResponse response = executeRequest(Optional.of(organizationDto), qgName);
+
+    assertThat(response.getName()).isEqualTo(qgName);
+    assertThat(response.getId()).isNotNull();
+    dbSession.commit();
+
+    QualityGateDto qualityGateDto = dbClient.qualityGateDao().selectByOrganizationAndName(dbSession, organizationDto, qgName);
     assertThat(qualityGateDto).isNotNull();
   }
 
+  @Test
+  public void throw_ForbiddenException_if_incorrect_organization() {
+    logInAsQualityGateAdmin(db.getDefaultOrganization());
+    OrganizationDto otherOrganization = organizationDbTester.insert();
+
+    expectedException.expect(ForbiddenException.class);
+    expectedException.expectMessage("Insufficient privileges");
+
+    executeRequest(Optional.of(otherOrganization), "Default");
+  }
+
   @Test
   public void throw_ForbiddenException_if_not_gate_administrator() {
     userSession.logIn();
 
     expectedException.expect(ForbiddenException.class);
     expectedException.expectMessage("Insufficient privileges");
 
-    executeRequest("Default");
+    executeRequest(Optional.empty(), "Default");
   }
 
   @Test
```

Comment:
This test should be renamed to "throw_ForbiddenException_if_not_gate_administrator" => default organization should not be mentioned as there's nothing specific to default org